src/GovUk.Education.ExploreEducationStatistics.Notifier/host.json

```diff
@@ -1,3 +1,8 @@
 {
-    "version": "2.0"
+    "version": "2.0",
+    "logging": {
+        "logLevel": {
```

Comment:
Mistake in `logLevel` here?

Reply:
no, functions host.js is slightly different in its formatting for some reason